overlord/snapstate/snapmgr.go

```diff
@@ -234,6 +235,7 @@ var ErrNoCurrent = errors.New("snap has no current revision")
 
 const (
 	errorOnBroken = 1 << iota
+	loadStore
```

Comment:
While `load` is clear the word `store` may be the counterpart of `load` or be the `snap store`. I don't have a better suggestion than `loadCachedStoreStuff` but perhaps you can come up with something.

Reply:
given that is passed to a read function not sure we need to repeat read,  just auxStoreInfo would be ok

Reply:
I can't have a constant called `auxStoreInfo` and a type called `auxStoreInfo` though

Reply:
sorry, you are right, single namespace and everything ....  withAuxStoreInfo ?  alsoAuxStoreInfo? (we could use a verb but then it becomes longish for no clear reason given context)